src/behaviors/my-schedule-behavior.html

```diff
@@ -43,8 +45,21 @@
           if (!this.app.user.sessions) {
             this.app.user.sessions = {};
           }
-          this.set('app.user.sessions.' + id, true);
-          HOVERBOARD.Elements.Template.$.toast.showMessage('{$ addedToSchedule $}');
+          this.set('app.user.sessions.' + id, true);    
+          
+          // [temp fix] to trigger app.sessions observer and re-filter my-schedule
+          // @ozasadnyy, I hope you won't have heart attack
+          // DevFest is coming :)
+          this.set('app.user.sessions.000', null);          
+          HOVERBOARD.Elements.Template.$.toast.showMessage('{$ addedToSchedule $}');     
+
+          if (isWorkshop) {
+            HOVERBOARD.Elements.Template.$.infoDialog.headerText = '{$ workshopWarning.headerText $}';
```

Comment:
Can we invoke open with options? Like:
```
HOVERBOARD.Elements.Template.$.infoDialog.open({
headerText : '',
infoText: '',
ctaButtonText: ''
})
```
?

Reply:
No, we can't, Iron-overlay-behaviour does not have such options when open dialog

Reply:
But `info-dialog` it's our component so we can define such function

Reply:
We can, but is it any difference?

Reply:
In this case, we will be able to use it later. But you are right, we are going to burn this website after the event 🔥